docs/content/learn/tutorials/component/_index.md

```diff
@@ -0,0 +1,335 @@
+A `component-definition` contains:
+
+- `metadata` (required) - Provides document metadata for the component definition. This is similar to metadata for other OSCAL models, therefor is not described extensively in this tutorial.
+- `import-component-definition` (optional) – Identifies component definition from other resources from which related information is referenced.  Use of `import-component-definition` is not covered in this tutorial.
+- `component` (optional) - Defines a given component in the component definition. Zero or more `component` objects may be used.
+- `capability` (optional) - Defines a given capability in the component definition. Zero or more `capability` objects may be used. Capabilities are not covered in this tutorial.
+- `back-matter` (optional) – Contains references used within the component definition. Use of `back-matter` is not covered in this tutorial.
+{{% /tab %}}
+{{% tab %}}
+{{< highlight yaml "linenos=table" >}}
+---
+component-definition:
+  uuid: a7ba800c-a432-44cd-9075-0862cd66da6b
+  metadata:
+  import-component-definition:
+  component:
+  capability:
+  back-matter:
+
+{{< /highlight >}}
+
+The root of the OSCAL component definition model is `component-definition`. The `@id` attribute (on line 3) is the document's universally unique identifier (UUID), a unique 128-bit number displayed as a string of hyphenated hexadecimal digits as defined by RFC 4122. OSCAL documents use a version 4 UUID (randomly generated) to uniquely identify the document.
+A `component-definition` contains:
+
+- `metadata` (required) - Provides document metadata for the component definition. This is similar to metadata for other OSCAL models, therefor is not described extensively in this tutorial.
+- `import-component-definition` (optional) – Identifies component definition from other resources from which related information is referenced.  Use of `import-component-definition` is not covered in this tutorial.
+- `component` (optional) - Defines a given component in the component definition. Zero or more `component` items may be used.
+- `capability` (optional) - Defines a given capability in the component definition. Zero or more `capability` items may be used. Capabilities are not covered in this tutorial.
+- `back-matter` (optional) – Contains references used within the component definition. Use of `back-matter` is not covered in this tutorial.
+{{% /tab %}}
+{{< /tabs >}}
+
+For simplicity of this tutorial, we will only discuss certain data structures in the sections that follow and identify how they can be used to represent our component definition.
+
+## Defining the Component Definition’s Metadata
```

Comment:
This title has a different structure than the "Component" "Protocol", etc. 
Maybe making the titles fr Metadata, Component, Protocol, etc a:
###  level 3 subsection as oppose to 
## level 2 which is the "Creating an OSCAL Component Definition Model" 
would help structuring the content.